autonomy/cli/publish.py

```diff
@@ -130,6 +133,7 @@ def publish_service_local(ctx: Context, public_id: PublicId) -> None:
     author_dir = Path(target_dir).parent
     if not os.path.exists(author_dir):
         os.makedirs(author_dir, exist_ok=True)
+    # TODO: also make services dir?
```

Comment:
not clear why we need to make an author dir, but not the services dir

Reply:
`os.makedirs` work recursively and creates missing parent directories as well

Reply:
So then why are we creating the one for the author?